packages/apollo-engine-reporting/src/treeBuilder.ts

```diff
@@ -16,7 +11,7 @@ export class EngineReportingTreeBuilder {
   public startHrTime?: [number, number];
   private stopped = false;
   private nodes = new Map<string, Trace.Node>([
-    [rootResponsePath, this.rootNode],
+    [responsePathAsString(undefined), this.rootNode],
```

Comment:
See comment below as to what this suggestion is suggesting.

```suggestion
    [responsePathAsString(), this.rootNode],
```

Reply:
I don't know the difference :)